webapp/src/components/loading.scss

```diff
@@ -0,0 +1,3 @@
+.loading-panel {
+    align-items: baseline !important;
```

Comment:
do we need !important here?

Reply:
Also, this should be centre aligned

Reply:
@raghavaggarwal2308 Mattermost's styling is overriding the styling provided by us hence using `!important` is necessary.
the existing styling with which the ellipses are not aligned with the text is `center-aligned`, using `baseline` fixes the issue.